tests/testthat/test-get-country-regional-data.R

```diff
@@ -0,0 +1,53 @@
+# Tests each dataset returned by countries specified in get_info_covidregionaldata
```

Comment:
context

Reply:
Is `context()` still needed? I was looking at: https://testthat.r-lib.org/reference/context.html where it is deprecated, and have tried to make test names descriptive enough to be used alone.

Reply:
maybe I am just old! Looks like you are correct.